src/config_entries.c

```diff
@@ -67,6 +67,36 @@ int git_config_entries_new(git_config_entries **out)
 	return error;
 }
 
+int git_config_entries_dup_entry(git_config_entries *entries, const git_config_entry *entry)
+{
+	git_config_entry *dup;
+	int error;
+
+	dup = git__calloc(1, sizeof(git_config_entry));
+	GIT_ERROR_CHECK_ALLOC(dup);
```

Comment:
Good catch. I'm surprised Coverity didn't complain, and noticed that it doesn't seem to be receiving updates ("May", it says)…

Reply:
Maybe this is related to our move to Azure Pipelines?

Reply:
🙁 It was definitely working at one point, then coverity went offline.  I'll take a look when I have a moment.